knowledge/astronomy/constellations/Phoenix/qna.txt

```diff
@@ -0,0 +1,19 @@
+created_by: 
+domain: constellation
+seed_examples:
+- answer: | The constellation does not lie on the galactic plane of the Milky Way, and there are no prominent star clusters
```

Comment:
Thank you for your submission; taking a look at your `qna.yaml`, we noticed that you have some issues with the multiline `yaml` syntax.

Can you take a look at: https://yaml-multiline.info/ and fix up the file? Thanks!